FWCore/MessageService/test/UnitTestClient_H.cc

```diff
@@ -14,8 +14,17 @@ void
                            , edm::EventSetup const & /*unused*/
                               )
 {
+#ifdef EDM_ML_DEBUG
```

Comment:
I had missed that `UnitTestClient_Hd.h` turned on debugging. So the `UnitTestClient_Hd.cc` is technically OK but please still remove `ifdef EDM_ML_DEBUG` since it is unnecessary. In fact, you can just remove the use of those variable all together and just use `true` and `false` in the `IfLog*` calls (same for`UnitTestClient_H.cc`) .

As for `UnitTestClient_H.cc`, at the top of the file do
```
#if defined EDM_ML_DEBUG
#undefine EDM_ML_DEBUG
#endif
```
That way even if this is compiled with debug the unit test will still give the same output and therefore will pass.